src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1972,7 +1972,8 @@ because xbuild doesn't support framework reference assemblies.
     DependsOnTargets="_ResolveAssemblies"
 >
   <ItemGroup>
-    <_AndroidResolvedSatellitePaths Include="@(ReferenceSatellitePaths)" />
+    <!-- In .NET 6+, the .NET SDK locates these files -->
```

Comment:
nitpick: Should this be .NET 8+ since we are not releasing this fix for .NEt 6?

Reply:
I think we should actually send this to 7, could also consider 6?

Reply:
Can we put stuff in 6 still?